src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -141,18 +143,37 @@ export class DictionaryConcepts extends Component {
     }));
   }
 
+  checkMembershipStatus(username) {
+    const {
+      match: {
+        params: {
+          type, typeName,
+        },
+      },
+    } = this.props;
+    if (type === 'orgs') {
+      const url = `/${type}/${typeName}/members/${username}/`;
+      this.props.fetchMemberStatus(url);
+    }
+  }
+
   render() {
     const {
       match: {
-        params: { typeName },
+        params: { type, typeName },
       },
       location: { pathname },
       concepts,
       filteredClass,
       filteredSources,
       loading,
+      userIsMember,
     } = this.props;
-    const username = typeName === getUsername();
+    const username = typeName === getUsername() || userIsMember;
+    const org = {
+      id: (type === 'orgs') ? typeName : '',
```

Comment:
`typeName` is being compared to the result of `getUsername` meaning it is a username and of type string. Assigning this to a variable name called `id` seems a bit of a misuse. Could you rename the variable name?

Reply:
Thanks for the feedback. However, I think `id` is the appropriate name because it is unique and also comes as `id` from the data that comes from the server.

Reply:
I have made the update.